### PR TITLE
Change references to v1alpha3 to v1

### PR DESCRIPTION
**Pull request by chrisnegus**
Signed-off-by: Christopher Negus <striker57@gmail.com>

This PR updates all v1alpha3 references to v1 in the user-guide.

Fixes: #601

@cwilkers

### CODE REVIEW

File: docs/operations/containerized_data_importer.md [hunk 1]
```diff
@@ -127,7 +127,7 @@ For example, trying to upload to the IP address 192.168.39.32 at port 31001 woul
 
      0 B / 193.89 MiB [-------------------------------------------------------]   0.00% 0s
 
-    Post https://192.168.39.32:31001/v1alpha1/upload: x509: cannot validate certificate for 192.168.39.32 because it doesn't contain any IP SANs
+    Post https://192.168.39.32:31001/v1/upload: x509: cannot validate certificate for 192.168.39.32 because it doesn't contain any IP SANs
```
Comment (awels) [inline docs/operations/containerized_data_importer.md:130]: This is CDI should be v1beta1

File: docs/operations/containerized_data_importer.md [hunk 2]
```diff
@@ -188,7 +188,7 @@ This happens because the cdi-uploadproxy certificate is self signed and the syst
 
      0 B / 193.89 MiB [-------------------------------------------------------]   0.00% 0s
 
-    Post https://cdi-uploadproxy:31001/v1alpha1/upload: x509: certificate signed by unknown authority
+    Post https://cdi-uploadproxy:31001/v1/upload: x509: certificate signed by unknown authority
```
Comment (awels) [inline docs/operations/containerized_data_importer.md:191]: This is CDI should be v1beta1

**Review by awels — APPROVED** [review]
/lgtm
/approve

**Review by cwilkers — APPROVED** [review]
/approve

File: docs/operations/containerized_data_importer.md (hunk 1 above)
Comment (chrisnegus) [reply docs/operations/containerized_data_importer.md]: Done.

File: docs/operations/containerized_data_importer.md (hunk 2 above)
Comment (chrisnegus) [reply docs/operations/containerized_data_importer.md]: Done.

**Pull request merged**